src/sprites.asm

```diff
@@ -777,60 +922,321 @@ RTS
 .koopanumbers
 db $04,$04,$05,$05,$07
 
-Sprite0D:		;keyhole
+Sprite0D:;Bob-omb
+JSR Sprite0F
```

Comment:
Instead of adding a call to Sprite0F, you should move the Sprite0D label to the same position as Sprite0F. Look at how Sprite08-Sprite0C do it immediately above.

Reply:
I know that I'm also fixing it ... it's just that I did not know about the sprites